include/grpcpp/impl/codegen/client_callback.h

```diff
@@ -174,6 +181,23 @@ class ClientBidiReactor {
   }
   void StartWritesDone() { stream_->WritesDone(); }
 
+  /// Holds are needed if this stream has operations that take place on it after
+  /// StartCall but from outside one of the reactions (OnReadDone, etc). If a
```

Comment:
Before "If a ", maybe add: "Holds must be added before calling StartCall()."

This is covered in the example portion, but it implies it is optional to do it before StartCall() while we want to make the stronger statement that they MUST do it before StartCall().